src/components/DisplayNames/DisplayNamesTooltipItem.tsx

```diff
@@ -1,50 +1,45 @@
-import PropTypes from 'prop-types';
-import React, {useCallback} from 'react';
+import React, {RefObject, useCallback} from 'react';
+import {Text as RNText, StyleProp, TextStyle} from 'react-native';
 import Text from '@components/Text';
 import UserDetailsTooltip from '@components/UserDetailsTooltip';
+import {AvatarSource} from '@libs/UserUtils';
 import styles from '@styles/styles';
 
-const propTypes = {
-    index: PropTypes.number,
+type DisplayNamesTooltipItemProps = {
+    index?: number;
 
     /** The full title of the DisplayNames component (not split up) */
```

Comment:
NAB: This comment is inaccurate.

Reply:
I'm going to merge now. If you would please do this in a follow up that would be great.